package.json

```diff
@@ -20,6 +20,7 @@
     "test:cli": "node tests/runner",
     "test:inspect": "node --inspect --debug-brk tests/runner",
     "test:packages": "node scripts/run-packages-spec.js",
+    "test:universal": "UNIVERSAL=true node tests/e2e_runner.js",
```

Comment:
I know, but I was not sure if you want to have the universal tests in the master branch. But now as we have a universal branch, I'll enable the test.